js/packages/common/src/utils/ids.ts

```diff
@@ -45,7 +45,7 @@ export interface PublicKeyStringAndAccount<T> {
   account: AccountInfo<T>;
 }
 
-export const WRAPPED_SOL_MINT = new PublicKey(
+export let WRAPPED_SOL_MINT = new PublicKey(
```

Comment:
should stay a const.

Reply:
Cannot assign to 'QUOTE_MINT' because it is a constant.ts(2588)